src/ResourceManager/Profile/Commands.Profile.Test/EnvironmentCmdletTests.cs

```diff
@@ -105,6 +105,7 @@ public void AddsAzureEnvironmentUsingARMEndpoint()
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ResourceManager], cmdlet.ResourceManagerEndpoint);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ActiveDirectory], cmdlet.ActiveDirectoryEndpoint);
             Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.ActiveDirectoryServiceEndpointResourceId], cmdlet.ActiveDirectoryServiceEndpointResourceId);
+            Assert.Equal(env.Endpoints[AzureEnvironment.Endpoint.GraphEndpointResourceId], cmdlet.GraphAudience);
```

Comment:
cmdlet.GraphAudience can never be null because RetrieveMetaDataEndpoints will throw an exception if it's null.